src/graphics/particle/MagicFlare.cpp

```diff
@@ -502,7 +502,7 @@ void FlareLine(Vec2f fromPos, Vec2f toPos, Entity * io) {
 			long z = Random::get(0, FLARELINERND);
 			z += FLARELINESTEP;
 			if(!io) {
-				z = long(z * g_sizeRatio.y);
+				z *= long(g_sizeRatio.y);
```

Comment:
This looks wrong, long(g_sizeRatio.y) is going to change abruptly from 1 (height 480 - 959) to 2 (height 960 to 1439) to 3 (1440p+) etc. instead of scaling the y offset smoothly.

Reply:
My bad. Removed.